Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM golang:1.18 AS builder
+ENV GOPROXY="https://goproxy.cn"
+WORKDIR /app
+COPY go.mod /app/go.mod
+COPY go.sum /app/go.sun
+RUN go mod download
```

Comment:
Why do we need to run `go mod download` separately?

Reply:
for reusing docker build cache.